src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");

```

Comment:
The message 'Truck started working' is not very informative. It would be better to indicate what type of work the Truck is performing to make the message more informative as per the checklist.